src-js/app.js

```diff
@@ -1,3 +1,77 @@
 import * as Cookie from 'js-cookie';
+import * as lodash from 'lodash.throttle';
 
 window.Cookie = Cookie;
+
+// Instant search initialization
+var search = instantsearch({
+  appId: '06F2EYGKB6',
+  apiKey: 'e80a0ac1e290d3b17ff1ea1692b94285', // search-only API Key; safe for front-end code
+  indexName: 'documentation',
+  routing: true
+});
+
+// initialize SearchBox
+search.addWidget(
+  instantsearch.widgets.searchBox({
+    container: '#search-box',
+    cssClasses: {
+      input: 'instantsearch-search'
+    },
+    placeholder: 'Search Documentation',
+    queryHook: lodash(function (query, searchFunction) { searchFunction(query); }, 1000)
+  })
+);
+
+// initialize SearchBox
+search.addWidget(
+  instantsearch.widgets.searchBox({
+    container: '#mobile-search-box',
+    cssClasses: {
+      input: 'instantsearch-search'
+    },
+    placeholder: 'Search Documentation'
+  })
+);
+
+// initialize hits widget
+search.addWidget(
+  instantsearch.widgets.hits({
+    container: '#instant-hits',
+    templates: {
+      empty: 'No results',
+      item: '<a href="/docs{{ url }}"><div class="result-item-wrap"><strong>{{ title }}</strong><br><p>{{ content }}</p></div>'
+    }
+  })
+);
+
+search.start();
+
+(function () {
```

Comment:
Why is this in an IIFE? I could be wrong here or be missing something about their lib, the impression I had is that IIFEs are good for making containing values so they don't get leaked onto the global scope. I think that's something we don't need to worry about since Webpack will put all of our code into modules. FWIW, I think it does that with IIFEs 😄